src/MobileApps/MyDriving/MyDriving.UWP/Views/CurrentTripView.xaml.cs

```diff
@@ -151,6 +153,18 @@ void OnPropertyChanged(object sender, PropertyChangedEventArgs e)
 
         private async Task StartTrackingAsync()
         {
+            // Request permission to access bluetooth
+            try
+            {
+                DeviceInformationCollection deviceInfoCollection =
```

Comment:
Can you check if this code block is hit when you put the app in background and got it back again ?  Also make sure when the app comes back from background it works properly.